_gcode/M605.md

```diff
@@ -2,7 +2,7 @@
 tag: m0605
 title: Multi Nozzle Mode
 brief: Set the behavior mode for a multiple nozzle setup
-author: thinkyhead
+author: thinkyhead | pacocatech
```

Comment:
You should add a new line and use `contrib:` instead:

`contrib: pacocatech`

Reply:
ok, i will ajust

Reply:
Requested adjustments made and committed.